src/Service.Tests/TestHelper.cs

```diff
@@ -237,5 +237,50 @@ public static void AddMissingEntitiesToConfig(RuntimeConfig config, string entit
             },
             ""entities"": {}" +
           "}";
+
+        /// <summary>
+        /// A valid REST API request body with correct parameter types for all the fields.
+        /// </summary>
+        public const string REQUEST_BODY_WITH_CORRECT_PARAM_TYPES = @"
```

Comment:
Why define these constants in the TestHelper.cs if they are only used in the ConfigurationTests.cs?